JAR missing on Maven Central for problem-spring-web-starter 0.25.0

## Description
When including the `problem-spring-web-starter` artifact in the Maven configuration, building a project fails since there is no JAR in Maven Central for this version:

```xml
<dependency>
  <groupId>org.zalando</groupId>
  <artifactId>problem-spring-web-starter</artifactId>
  <version>0.25.0</version>
</dependency>
```

```
$ mvn clean install
...
Downloading from central: https://repo.maven.apache.org/maven2/org/zalando/problem-spring-web-starter/0.25.0/problem-spring-web-starter-0.25.0.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11.565 s
[INFO] Finished at: 2019-10-14T15:05:48+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project *myproject*: Could not resolve dependencies for project *somegroup*:*myproject*:jar:3.2.0-SNAPSHOT: Could not find artifact org.zalando:problem-spring-web-starter:jar:0.25.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```


## Expected Behavior
There should be a JAR for this version on Maven Central.

## Actual Behavior
There is not JAR on Maven Central for this version, only a POM:
https://repo1.maven.org/maven2/org/zalando/problem-spring-web-starter/0.25.0/

## Possible Fix
Deploy a JAR to Maven Central?

--- a/problem-spring-web-starter/pom.xml
+++ b/problem-spring-web-starter/pom.xml
@@ -10,18 +10,11 @@
     </parent>
 
     <artifactId>problem-spring-web-starter</artifactId>
-    <packaging>pom</packaging>
 
     <name>Problem: Spring Web Starter</name>
     <description>Spring starter module for Problem Spring Web</description>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
-            <version>${spring-boot.version}</version>
-        </dependency>
-
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>problem-spring-web-autoconfigure</artifactId>

